Add link to set_verbosity documentation (#1950)
nest/nest-simulator: doc/userdoc/getting_started.rst

**And that's it! You have performed your first neuronal simulation in NEST!**

Need a quieter NEST?
--------------------

Take a look at the `set_verbosity <https://nest-simulator.readthedocs.io/en/latest/ref_material/pynest_apis.html#nest.lib.hl_api_info.set_verbosity>`_

> documentation, which describes how to display less warning messages.

Reviewer: ```suggestion
documentation, which describes how to display fewer messages on the terminal.
```
